applications/portfolioDrafts/deletePortfolioDraft.ts

```diff
@@ -0,0 +1,57 @@
+import { DeleteCommand, DeleteCommandInput } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { dynamodbClient as client } from "./utils/dynamodb";
+import { ErrorCodes } from "./models/Error";
+import { ErrorResponse, ErrorStatusCode, NoContentResponse } from "./utils/response";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+
+/**
+ * Deletes a Portfolio Draft
+ *
+ * @param event - The DELETE request from API Gateway
+ */
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+
+  if (!portfolioDraftId) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "PortfolioDraftId must be specified in the URL path" },
+      ErrorStatusCode.BAD_REQUEST
+    );
+  }
+
+  if (event.body) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "HTTP request body must be empty" },
+      ErrorStatusCode.BAD_REQUEST
+    );
+  }
```

Comment:
Do we actually want to _reject_ requests with a body? Or just choose to ignore them? I can see an argument either way. The specification is unclear as it doesn't define a body (where as `POST /portfolioDrafts` explicitly specifies it _must_ be empty). Should we decide on a convention here?

I like the strategy of being liberal in what we accept and conservative in what we emit; so with that since the spec doesn't specify a body schema, we should either:
- update the spec to forbid non-empty bodies
- accept and ignore the body

And we may want to consider the possibility that we _could_ actually accept parameters to the `DELETE` operation in the future.

Reply:
I could be convinced that we should be liberal in what we accept.  The DELETE in the spec does not specify a requestBody.

Reply:
I think it's just copy-paste from `POST /portfolioDrafts`.  Removing.

Reply:
Fix in https://github.com/dod-ccpo/atat-web-api/pull/40/commits/c9487c1f822a01d4b945c5165b237515be2bc6ec